File: python/ram.py
from komponen import Komponen

class Ram(Komponen):
    def __init__(self, kapasitas_gb: int, ddr: str, merk: str, nama: str):
        super().__init__(merk, nama)
        self.kapasitas_gb = kapasitas_gb
        self.ddr = ddr

    def info(self):
        print(f"RAM: {self.merk} {self.nama} ({self.kapasitas_gb} GB, {self.ddr})")

File: java/Ram.java
class Ram extends Komponen {
  private int kapasitasGB;
  private String ddr;

  public Ram(int kapasitasGB, String ddr, String merk, String nama) {
      super(merk, nama);
      this.kapasitasGB = kapasitasGB;
      this.ddr = ddr;
  }

  @Override
  public void info() {
      System.out.println("RAM: " + merk + " " + nama + " (" + kapasitasGB + " GB, " + ddr + ")");
  }
}
